data/presets/amenity/stripclub.json

```diff
@@ -0,0 +1,40 @@
+{
+    "icon": "fas-place-of-worship",
```

Comment:
I don't think [this icon](https://fontawesome.com/icons/place-of-worship?s=solid&f=classic) is a good match for this tag.

Reply:
Looks like a strip club to me. Big door, no windows, big grand entrance. The icon itself is a pretty close match.

![stripclub](https://user-images.githubusercontent.com/19922/203165586-bab22f19-d8a5-45fb-b85e-a84c87b5fb60.jpg)

What is your reservation? Is there a better suggestion?

Reply:
We’re trying not to reuse preset icons so much, especially for quite unrelated features: #41. Sounds like you’re aware of the need for something more specific: https://github.com/FortAwesome/Font-Awesome/issues/8254#issuecomment-1321023839. The icon can come from [a few sources besides FontAwesome](https://github.com/ideditor/schema-builder/#icon), including a custom-drawn icon if you prefer to draw something yourself.

Reply:
I'd just use `temaki-money_hand` until something better comes along.

<img src="https://raw.githubusercontent.com/ideditor/temaki/main/icons/money_hand.svg" alt="temaki-money_hand" width="40" height="40">

Reply:
`temaki-money_hand` doesn't look at all like a strip club. Normally, afaik, you don't hand strippers money either. You either (a) make it rain, or (b) affix the money to the stripper. Or so I've read.

But more important `temaki-money_hand` would also be reused.

* https://github.com/openstreetmap/id-tagging-schema/search?q=temaki-money_hand

So the question remains, what icon should I use? I have 0 artistic skill.

Reply:
> temaki-money_hand doesn't look at all like a strip club. 

And your strip club above looks like a museum.

<img src="https://raw.githubusercontent.com/mapbox/maki/main/icons/museum.svg" alt="maki-museum" width="40" height="40">

Reply:
Make an icon that looks like [XXX]. Simple.

Reply:
Far too detailed; doesn’t scale well to the _very_ small sizes that these icons can appear in. These icons will be shown to everyone, not just those actively seeking out strip clubs, in the event that someone encounters one that’s already mapped. They’ll need to have some general idea of what it means independently of the preset name.

Do you have an objection to my suggestion of a simple text icon? If you’re unable to create an image that consists of the string “XXX”, then someone else can create it for you.

Reply:
Does this one work? https://commons.wikimedia.org/wiki/File:Pornproject_logo.svg

Reply:
Something along those lines, yes, but it would need to be in black and white, similar to the other icons. (Alternatively, there’s already [an “X” icon for railroad crossings](https://ideditor.github.io/temaki/docs/#icon-temaki-x_oblique) that could be modified for this purpose.) [Temaki](https://github.com/ideditor/temaki/) would be the place to submit a new icon.

Reply:
Yes, I think either your suggestion (with the color modification I mentioned) or @tyrasd’s suggestion could work. I just didn’t want us to overthink it. You’re right that many of the icons aren’t unique yet – it’s something we’re working towards, but new presets should at least avoid reusing an icon that’s completely unrelated. We’ve already have several instances where FontAwesome dramatically changes the look of an icon while keeping it semantically consistent with the file name.

Reply:
I can also open the request in Temaki for a new icon. Not being a hardass on the art stuff either. I just suck with the open source software on it. I spent 2 hours trying to figure out how to draw a box, asked a question on it, and 12 years later its since helped 150k people: [How do I draw a box in GIMP?](https://superuser.com/questions/167873/how-do-i-draw-a-box-in-gimp). I still have to google for how to draw a box in GIMP. GIMP is terrible, and I don't have a VM with photoshop. =(

Reply:
Ah, I feel you. I find vector drawing in Inkscape to be much more intuitive, and as it turns out, projects like Temaki and FontAwesome require vector graphics in SVG format.

Reply:
I've actually only ever heard good things about Inkscape, I'll check it out. It's open source. As long as it's not gimp.